app/models/user.rb

```diff
@@ -15,4 +18,16 @@ def normalize_queue_item_list
   def queued_video?(video)
     queue_items.find_by(video: video)
   end
+
+  def already_followed?(leader)
+    following_relationships.map(&:leader).include?(leader)
+  end
+
+  def is_the?(role)

```

Comment:
:). Yep, i know, this name really sucks. Sometimes just cant find a name that i like.